app/src/userpages/components/ProductsPage/Members/index.jsx

```diff
@@ -170,10 +170,14 @@ const Members = () => {
                 })
             }
             selection.none()
+
+            if (loadMembersRef.current) {
+                loadMembersRef.current()
```

Comment:
I see! This piece of code pretty much reloads the member list using current filters, etc. Same thing happens on approval (a reload). Slowly getting there…

I guess we're gonna have to add `isMounted()` in both of them callbacks after the `await` loops.

Reply:
This change fixes the original issue.

Reply:
Would be better if backend supported deleting multiple entries at the same time but this works for now... let's see how it behaves when we are >100 or even >1000 requests 🙈

Reply:
I hear you. I hear you loud and clear.